framework/doc/content/source/kernels/KernelScalarBase.md

```diff
@@ -0,0 +1,6 @@
+# KernelScalarBase
+
+Mention AD version provided, and with warning for global only.
+describe the parameters like the kernel system
+say what loops are added, primary and other scalars
+
```

Comment:
This looks incomplete?

Reply:
Will modify next round; have started it in the latest commit.

Reply:
I (finally) finished writing the documentation of the base class for kernels. I will go to the mortar system next.